geometry_msgs/PolygonStamped shape
std_msgs/String name
---
std_msgs/Bool outcome
